apps/nextjs-website/src/lib/header.ts

```diff
@@ -40,7 +40,9 @@ export const makeHeaderProps = (
   ...(header.data.attributes.avatar &&
     header.data.attributes.avatar.data && {
       avatar: {
-        src: header.data.attributes.avatar.data.attributes.url,
+        src:
+          'http://localhost:1337' +
```

Comment:
Same as [this](https://github.com/pagopa/b2b-portals/pull/83#discussion_r1440642501)